### ISSUE TITLE
Add frontend for each example

### ISSUE THREAD

**Issue by cmichi** [paritytech/ink-examples#32]
Tracking issue to reflect this goal on the Parity Roadmap.

**Comment by niklasp** [comment]
how is it going with this? i could help out if still some parts are missing.

**Comment by cmichi** [comment]
@niklasp Thanks for writing in here! Adding more frontend examples is currently low on our list of priorities, but we'll have some other suggestions coming your way shortly!

**Comment by cmichi** [comment]
@niklasp This one could be interesting for you: https://github.com/use-ink/ink-alliance/blob/main/bounties/001-website_upgrade.md. Would be great to see you apply for it!

**Comment by niklasp** [comment]
Yes i have seen it but the scope is much larger isnt it? And also the bounty does not include this issue?

I am considering an application but the time horizon is quite short. I will try to find interested  (ux)designers
